src/fidesops/task/graph_task.py

```diff
@@ -72,6 +73,13 @@ def result(*args: Any, **kwargs: Any) -> List[Optional[Row]]:
                         self.log_start(action_type)
                     # Run access or erasure request
                     return func(*args, **kwargs)
+                except PrivacyRequestPaused as ex:
+                    logger.warning(
+                        f"Privacy request {method_name} paused {self.traversal_node.address}"
+                    )
+                    self.log_paused(action_type, ex)
+                    # Re-raise to stop privacy request execution.
+                    raise
```

Comment:
This is a primary change.  

It's worth pointing out that in our current design, if a node fails three times on retry, we catch that final exception, and just pass an empty list or a 0 downstream, still allowing the request to complete.  The Pause feature (and soon, the Restart after Exception feature) actually stops execution, cancelling other remaining tasks, rather than sending dummy values downstream.

Reply:
As I understand it the pause feature and auto retry feature are distinct. Am I right in thinking that pause on error will only pause after the configured auto retries are exhausted?

Reply:
they are different features, but they interact because auto-retry catches the first paused exception here. And no, pause re-raises here, so it doesn't automatically retry.  If a node was just paused, you wouldn't have any "retry" execution logs created for it.

Reply:
unlike the "pause" feature, in which I don't have any retries, for the similar feature to "restart after failure", I'm thinking we still retry for the configured amount of times in case it is a temporary glitch, and then re-raise the error the final time, cancelling the remaining tasks, rather than sending the zero values downstream.